.github/workflows/datamanager.yml

```diff
@@ -13,13 +13,20 @@ on:
       - "doc/**"
 
 env:
-  ENV_FILE: environment.yml
   PANDAS_CI: 1
+  PYTEST_WORKERS: "auto"
```

Comment:
The name of this file changed.

Reply:
I'm happy to revert that typo fix if you don't like it.

Reply:
Ah right looks like the original was a typo. Looks fine as is